### PR TITLE
refactor,chore(egen) : refcatored code according to template guidelines and added cleanup code

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes the following updates to the `automl_tabular_classification_beans.ipynb` notebook:
  Adds colab enterprise link.
  added rm $PIPELINE_FILE_NAME in cleanup 
  Updates some explanations for cells.
  Removes IS_TESTING condition in clean-up.
  Replaces REGION with LOCATION.
  Follows the new notebook template.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 1]
```diff
@@ -76,15 +79,15 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you train a scikit-learn tabular classification model and create batch prediction job for it through a Vertex AI pipeline using `google_cloud_pipeline_components`. The source and destination data for the batch prediction job is served in BigQuery.\n",
+        "In this tutorial, you train a scikit-learn tabular classification model and create batch prediction job for it through a Vertex AI pipeline using google_cloud_pipeline_components. The source and destination data for the batch prediction job is served in BigQuery.\n",
```
Comment (katiemn) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:82]: modify to: "create a batch prediction job..."

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 2]
```diff
@@ -76,15 +79,15 @@
...
-        "- Vertex AI `Pipelines`\n",
-        "- Vertex AI `Datasets`\n",
-        "- Vertex AI `Training`\n",
-        "- Vertex AI `Model Registry`\n",
-        "- Vertex AI `Batch Predictions`\n",
+        "- Vertex AI Pipelines\n",
+        "- Vertex AI Datasets\n",
+        "- Vertex AI Training\n",
+        "- Vertex AI Model Registry\n",
+        "- Vertex AI Batch Predictions\n",
```
Comment (katiemn) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:90]: lowercase: batch prediction

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 3]
```diff
@@ -76,15 +79,15 @@
...
         "\n",
         "This tutorial uses the following Google Cloud ML services and resources:\n",
         "\n",
-        "- Vertex AI `Pipelines`\n",
-        "- Vertex AI `Datasets`\n",
-        "- Vertex AI `Training`\n",
-        "- Vertex AI `Model Registry`\n",
-        "- Vertex AI `Batch Predictions`\n",
+        "- Vertex AI Pipelines\n",
+        "- Vertex AI Datasets\n",
```
Comment (katiemn) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:87]: Datasets -> dataset

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 4]
```diff
@@ -687,9 +574,9 @@
       },
       "source": [
         "### Create the trainer task\n",
-        "Within `trainer/`, create a module named `task.py` that serves as the entrypoint for your training code.\n",
+        "Within trainer/, create a module named task.py that serves as the entrypoint for your training code.\n",
         "\n",
-        "The trainer code below preprocesses the train set and store the preprocessing transforms in a scikit-learn pipeline. Further, a [Random Forest model is trained](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html) on the preprocessed train data and added as an estimator to the pipeline. After saving the model, it is uploaded to the Cloud Storage bucket for deployment.\n",
+        "The trainer code below preprocesses the train set and store the preprocessing transforms in a scikit-learn pipeline. Further, a [Random Forest model is trained](https://scikitlearn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html) on the preprocessed train data and added as an estimator to the pipeline. After saving the model, it's uploaded to the Cloud Storage bucket for deployment.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:579]: "store" -> "stores":

- The trainer code below preprocesses the train set and stores the preprocessing transforms in a scikit-learn pipeline.

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 5]
```diff
@@ -1074,25 +961,25 @@
...
+        "- location: Region where the pipeline needs to run.\n",
+        "- dataset_display_name: Display name for the managed dataset resource in Vertex AI.\n",
+        "- dataset_bq_source: BigQuery table URI to serve as a source for the managed dataset in Vertex AI.\n",
+        "- training_job_dispaly_name: Display name for the the custom python package training job.\n",
+        "- gcs_staging_directory: Staging directory for Vertex AI to store training artifacts.\n",
+        "- python_package_gcs_uri: Cloud Storage path to the Python package for training.\n",
+        "- python_package_module_name: Module name (trainer task) inside the Python package for training.\n",
+        "- training_split: Percentage of the total data to be considered for training.\n",
+        "- test_split: Percentage of the total data to be considered for testing. Split percentage parameters provided for the **CustomPythonPackageTrainingJobRunOp** component should always sum up to 1.\n",
+        "- training_container_uri: Pre-built container image URI for training the model. \n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:974]: Pre-built -> Prebuilt (remove hypen)

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 6]
```diff
@@ -1074,25 +961,25 @@
...
+        "- serving_container_uri: Pre-built container image URI for serving the model on Vertex AI.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:975]: Pre-built -> Prebuilt (remove hypen)

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 7]
```diff
@@ -928,7 +815,7 @@
       "source": [
         "### Copy the source distribution to Cloud Storage\n",
         "\n",
-        "To train the custom classification model using a pre-built container, copy the source distribution of your training application to a Cloud Storage path. While training you let the Vertex AI SDK locate the package through the `python_package_gcs_uri` parameter."
+        "To train the custom classification model using a pre-built container, copy the source distribution of your training application to a Cloud Storage path. While training you let the Vertex AI SDK locate the package through the python_package_gcs_uri parameter."
```
Comment (kittyabs) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:818]: Pre-built -> Prebuilt (remove hypen)

**Review by kittyabs — COMMENTED** [review]
some suggested edits

File: notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb [hunk 8]
```diff
@@ -172,201 +175,87 @@
...
         "\n",
-        "2. [Make sure that billing is enabled for your project](https://cloud.google.com/billing/docs/how-to/modify-project).\n",
+        "if \"google.colab\" in sys.modules:\n",
         "\n",
-        "3. [Enable the Vertex AI API]\n",
+        "    import IPython\n",
         "\n",
-        "4. If you are running this notebook locally, you need to install the [Cloud SDK](https://cloud.google.com/sdk)."
+        "    app = IPython.Application.instance()\n",
+        "    app.kernel.do_shutdown(True)"
```
Comment (katiemn) [inline notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb:208]: Please add the restart warning message from the notebook template

**Review by katiemn — APPROVED** [review]

**Pull request merged**